src/layout/FeaturesSection/FeatureShowcase.svelte

```diff
@@ -19,4 +19,4 @@
 
 <style global lang="scss">
 	@use "./FeatureShowcase";
-</style>
\ No newline at end of file
+</style>
```

Comment:
Sorry about this, i can't figure out how to revert it and i don't find it worth more commits to fix.